pkg/gofr/datasource/pubsub/eventhub/eventhub.go

```diff
@@ -47,20 +49,20 @@ type Client struct {
 	checkPoint *checkpoints.BlobStore
 	// processorCtx is being stored such that to gracefully shutting down the application.
 	processorCtx context.CancelFunc
-	cfg          Config
+	cfg          *Config
 	logger       Logger
 	metrics      Metrics
 	tracer       trace.Tracer
 }
 
-// New Creates the client for Eventhub
-func New(cfg Config) *Client {
+// New Creates the client for Eventhub.
+func New(cfg *Config) *Client {
```

Comment:
This change is done to get rid of the linter warning `hugeParam: cfg is heavy (128 bytes); consider passing it by pointer`

Reply:
The point has been raised also here

https://github.com/gofr-dev/gofr/pull/1245#discussion_r1861056964

Reply:
sure added `nolint` to avoid that linter warning